mrp_packaging/models/mrp_production.py

```diff
@@ -116,10 +114,10 @@ def create_mo_from_packaging_operation(self):
                     new_op.manual_production_lot = line.lot.name
                     continue
             for link_product in linked_raw_products:
-                if link_product['product_id'] in prod_line_ids:
-                    new_op.write(
-                        {'product_lines': [(1, link_product['product_id'],
-                                            link_product)]})
+                recs = new_op.product_lines.filtered(
+                    lambda x: x.product_id.id == link_product['product_id'])
+                if recs:
+                    recs.write(link_product)
```

Comment:
A ver...intento ser lo más clara posible..
La función lo que hace es crear OF para los envasados definidos en una OF de 'producto normal'.
Para ello, por cada paquete definido crea la OF, con sus datos correspondientes y calcula los datos. 
Después de haber calculado los datos, recorre los atributos del paquete que se va a producir, si alguno de estos tiene el campo raw_product relleno, genera un diccionario mediante una función para el producto definido en ese campo. Y luego, busca en los productos planificados de la nueva OF si hay ese producto, y si hay, le carga esos datos.
Lo que se ve en el código este que estamos mirando es la parte de escribir los datos en los productos planificados.